src/core/connection.h

```diff
@@ -463,6 +463,21 @@ typedef struct QUIC_CONNECTION {
     //
     uint64_t NextRecvAckFreqSeqNum;
 
+    //
+    // The maximum number of packets that can be out of order before an immediate
+    // acknowledgment (ACK) is triggered. If no specific instructions (ACK_FREQUENCY
+    // frames) are received from the peer, the receiver will immediately acknowledge
+    // any out-of-order packets, which means the default value is 1. A value of 0
+    // means out-of-order packets do not trigger an immediate ACK.
+    //
+    uint8_t ReorderingThreshold;
```

Comment:
Put these up after the other `uint8_t` (`SendAckFreqSeqNum`) so there isn't so much extra padding

Reply:
Didnt get you. I only see uint64_t SendAckFreqSeqNum;

Reply:
Put these before `SendAckFreqSeqNum`, next to the other `uint8_t` fields.